### ISSUE TITLE
project creation with license=none fails

### ISSUE THREAD

**Issue by ksilin** [sbt/sbt-fresh#44]
creating a fresh project with the default license settings works fine, creating one without a license fails reproducibly.

v 1.6.6, empty global build.sbt

```
> fresh license=none
java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:313)
	at scala.None$.get(Option.scala:311)
	at de.heikoseeberger.sbtfresh.Template$.readme(Template.scala:260)
	at de.heikoseeberger.sbtfresh.Fresh.writeReadme(Fresh.scala:67)
	at de.heikoseeberger.sbtfresh.FreshPlugin$.de$heikoseeberger$sbtfresh$FreshPlugin$$effect(FreshPlugin.scala:93)
	at de.heikoseeberger.sbtfresh.FreshPlugin$$anonfun$de$heikoseeberger$sbtfresh$FreshPlugin$$freshCommand$2.apply(FreshPlugin.scala:60)
	at de.heikoseeberger.sbtfresh.FreshPlugin$$anonfun$de$heikoseeberger$sbtfresh$FreshPlugin$$freshCommand$2.apply(FreshPlugin.scala:60)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$.process(Command.scala:93)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.State$$anon$1.process(State.scala:184)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.MainLoop$.next(MainLoop.scala:96)
	at sbt.MainLoop$.run(MainLoop.scala:89)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:68)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:63)
	at sbt.Using.apply(Using.scala:24)
	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:63)
	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:46)
	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:30)
	at sbt.MainLoop$.runLogged(MainLoop.scala:22)
	at sbt.StandardMain$.runManaged(Main.scala:54)
	at sbt.xMain.run(Main.scala:29)
```

**Comment by hseeberger** [comment]
Thanks! The whole license thing is a mess ;-)

**Comment by hseeberger** [comment]
Closed by #45
